include/deal.II/numerics/vector_tools_constraints.templates.h

```diff
@@ -529,26 +670,29 @@ namespace VectorTools
                     internal::VectorDoFTuple<dim> vector_dofs;
                     vector_dofs.dof_indices[0] = face_dofs[i];
```

Comment:
do you still need this line? Can you move this and the line above as far down in the code as possible (to the first use)